website/status_screen/templates/status_screen/venue_status_screen.html

```diff
@@ -0,0 +1,275 @@
+                    <a class="navbar-brand drop-out-header" href="/"><img
+                            src="{% static 'tosti/svg/TOSTI-logo.svg' %}"
+                            height="120"/></a>
+                </div>
+        </div>
+    </nav>
+{% endblock %}
+
+{% block page %}
+    <div class="container-fluid mt-3" id="status-screen-container">
+        <template v-if="active_shift">
+            <div class="row">
+                <div class="col-md-6 p-2">
+                    <h2>In progress</h2>
+                    <transition-group name="user_objects_with_orders_in_progress" tag="ul" class="user-order-list orders-in-progress px-5">
+                        <li v-for="(user_orders_object, user_id) in orders_in_progress_grouped_user" :key="`orders_in_progress_user_${user_id}`" class="user-order-item">
+                            <div class="order-user-name">${user_orders_object.user.first_name}$</div>
+                            <ul class="order-list">
+                                <li v-for="order in user_orders_object['orders']" :key="`user_${user_id}_order_${order.id}`" class="order-item">
+                                    <i v-if="order.product.icon !== null" :class="`fa-solid fa-${order.product.icon} me-2`"></i>
+                                    <i v-else class="fa-solid fa-question"></i>
+                                </li>
+                            </ul>
+                        </li>
+                    </transition-group>
+                </div>
+                <div class="col-md-6 p-2">
+                    <h2>Ready</h2>
+                    <transition-group name="orders_ready" tag="ul" class="user-order-list orders-ready px-5">
+                        <li v-for="(user_orders_object, user_id) in orders_ready_grouped_user" :key="`orders_ready_user_${user_id}`" class="user-order-item">
+                            <div class="order-user-name">${user_orders_object.user.first_name}$</div>
+                            <ul class="order-list">
+                                <li v-for="order in user_orders_object['orders']" :key="`user_${user_id}_order_${order.id}`" class="order-item">
+                                    <i v-if="order.product.icon !== null" :class="`fa-solid fa-${order.product.icon} me-2`"></i>
+                                    <i v-else class="fa-solid fa-question"></i>
+                                </li>
+                            </ul>
+                        </li>
+                    </transition-group>
+                </div>
+            </div>
+        </template>
+    </div>
+
+
+    <div id="player-container" style="display: none">
+        {% if player is not None %}
+            <div class="player w-75 mt-5 m-auto display-5">
+                <h1 class="mt-5">Currently playing:</h1>
+                {% render_player player %}
+            </div>
+        {% else %}
+            <h1 class="mt-5">No player for this venue</h1>
+        {% endif %}
+    </div>
```

Comment:
You don't have to do this with `display: none`, you can show this with another template and then adding `v-else` to the `v-if="active-shift"` container above.

Reply:
I tried that but it gives me an error because we are already mounting a vue app on the container above, that's why I used this hack